content/release-notes/1.17.0.md

```diff
@@ -0,0 +1,44 @@
+---
+date: 2020-07-13T08:00:00-07:00
+linktitle: "1.17.0"
+title: "1.17.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* GitOps - Added Github Enterprise support.
+* Snapshots - Added a "Internal Storage (Default)" option to the providers list for KURL embedded clusters. Which will switch back to the default configuration that velero was set up with when the app was installed.
+* Added `when` field support for config groups.
+{{</features>}}
+
+{{<changes>}}
+* Kotsadm - New Support bundles will be shown at the top of the list instead of the bottom.
+* Kotsadm - Showing warnings on the Version History page in the Admin Console when installing an application with invalid YAML.
+* Kotsadm - Added a page title for the 'View files' and 'Config' tabs.
+* Removed the KOTS metadata that was added to all application manifests. Which included the local application and the version sequence injected as annotations to all deployed manifests.
+* Show the airgap upload progress percentage more granularly.
+* Removed the "Re-analyze" button from the troubleshoot support bundle analysis page.
+* Auto deploy the app's first version when installing the app and no preflight checks are available or preflight checks pass.
+* Update to import k8s 1.18 libraries.
+* Show a descriptive error message when `kubectl kots download` is called with an app slug that does not exist.
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug in KOTS CLI where using `--deploy` flag when checking for updates does not deploy the latest release.
+* Fixed a bug in KOTS CLI where automated installs fail to deploy if no preflight checks are available.
+* Fixed a bug in KOTS CLI where checking for updates shows `There are no application updates available` even when updates are actually available.
+* Fixed an issue where license expiration isn't checked until after fetching application.
+* Fixed a bug where preflight checks were not triggered for airgap or automated installs.
+* Fixed a bug where Snapshots scheduled `@daily` run just on day 1 of the month instead of every day.
+* Fixed an issue in Kotsadm troubleshoot "View Files" tab that shows full path instead of just subdirectory.
+* Fixed an issue where the minio image included a binary with CAP_CHOWN set.
+* Fixed button styles on TLS page on KOTS cluster installs.
+* Fixed snapshot hooks status.
+* Fixed backup size on snapshots list.
```

Comment:
Is this when the backup size incorrectly says `0 bytes` or `undefined` or something like that? 

If so, I'd explicitly state that.